internal/token/fileset.go

```diff
@@ -4,6 +4,16 @@ import (
 	"github.com/influxdata/flux/ast"
 )
 
+type FileSet struct {
+	files []*File
+}
+
+func (f *FileSet) AddFile(filename string, size int) *File {
```

Comment:
It might be reasonable to wrap around the `go/token` package again. The intention of `FileSet` is that a `token.Pos` can be used to lookup which file it came from and, since I took this API from `go/token`, I think it would be helpful not to change that.

But, for the sake of this PR, I do not think this is urgent and so it doesn't need to be changed in this PR.

Reply:
This makes sense, I mimiced that API but left the implementation hollow. I think a separate PR to get FileSet and File all working makes sense here.